src/Display/Tabs/Tabs.tsx

```diff
@@ -112,12 +109,12 @@ Tabs.Pane = TabPane;
 
 interface Props {
   children: React.ReactNode;
-  variant?: EHorizontalTabVariant;
+  variant?: string;
```

Comment:
@ritikamotwani could you help experiment the following?
```
type Variants = keyof typeof EHorizontalTabVariant;
variant?: Variants;
```
I was thinking if we can have `variant? 'underlined' | 'colored'`. It might help us throw the error when we pass a wrong `variant` value in other projects.

Reply:
Oh I think `variant? 'underlined' | 'colored'` would be better

Reply:
Updated with `variant?: 'underlined' | 'colored'`
It would throw an error, if the value is incorrect for variant, theme, alignment

Reply:
I thought `type Variants = keyof typeof EHorizontalTabVariant;` equals `'underlined' | 'colored'`. So we can just need to update one place (enum) when we have a new value.

Btw, I think we can still have `type Variant = ''underlined' | 'colored'` and `variants? Variant` instead of setting `varaints? 'underlined' | 'colored'` everywhere.

Reply:
The reason I didn't use this is, when I write this it becomes `"UNDERLINED" | "COLORED"`, and then if I try using `typeof ETabAlignment[keyof typeof ETabAlignment]` it's the same as ETabVariant type (https://stackoverflow.com/questions/53662208/types-from-both-keys-and-values-of-object-in-typescript)

Reply:
Ohhh right, it is `keyof` 😅 , then what do you think of using `type Variant = ''underlined' | 'colored'`?

Reply:
Yes, that's much better! Thanks